railties/lib/rails/application.rb

```diff
@@ -141,6 +141,8 @@ def initialize(initial_variable_values = {}, &block)
       @reloader          = Class.new(ActiveSupport::Reloader)
       @reloader.executor = @executor
 
+      @url               = nil
```

Comment:
Don’t need this, no? Doesn’t seem to be here for consistency’s sake either.

Reply:
Its for consistency, see other similar vars on this class initialization.